math/mod.hlsl

```diff
@@ -11,6 +12,10 @@ license: |
 
 #ifndef FNC_MOD
 #define FNC_MOD
+float2 mod(float x, float y) {
```

Comment:
I think there is a typo here. This one should return `float` and not `float2`. Right? 
Thank you so much for contributing!

Reply:
Oh whoops, thanks 😅 will fix.

Reply:
Done